src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 0 36px;
+    text-align: center;
+  }
+
+  &__photo {
+    height: 134px;
+  }
+
+  &__info {
+    margin-inline: 16px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $second-text-color;
```

Comment:
Variables should be used for all colors. Ensure that the variable $second-text-color is defined in the `variables.scss` file and contains the correct value.